styles/_basecoat-footer.scss

```diff
@@ -35,3 +35,18 @@
     float: right;
   }
 }
+
+// swap social links and nav for right-to-left languages
+html[dir="rtl"] {
+  .footer-nav {  
+    @include breakpoint(md) {
+      float: right;
```

Comment:
@zeke while this is working good, I noticed few margin/padding visual bugs  related to RTL. There  are still things to be flipped in the styles.

RTLCSS post-processing could be quick solution for large style codebase. But it may need few restriction on input styles (like few overrides that shouldn't flipped in RTL output).

I would recommend using global variables. like  script-start: left/LTR | right/RTL & script-end: right/LTR | left/RTL . Then change all style to use variable.

Then later when CSS4 is released, which probably will ship _logical positioning_. you could easily switch to them then.

To give you full picture AFAIK, HTML `dir="auto"` is nice but partial solution, because most browsers didn't implement `:dir()` pseudo-class for CSS.